lambdas/tests/unit/handlers/test_document_reference_search_handler.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from unittest.mock import patch
+
+import pytest
+from botocore.exceptions import ClientError
+
+from handlers.document_reference_search_handler import lambda_handler
+from tests.unit.helpers.dynamo_responses import EXPECTED_RESPONSE, MOCK_RESPONSE, MOCK_EMPTY_RESPONSE, UNEXPECTED_RESPONSE
+from services.dynamo_query_service import DynamoQueryService
+from utils.lambda_response import ApiGatewayResponse
+
+
+@pytest.fixture
+def valid_nhs_id_event():
+    api_gateway_proxy_event = {
+        "queryStringParameters": {"patientId": "9000000009"},
+    }
+    return api_gateway_proxy_event
+
+
+@pytest.fixture
+def invalid_nhs_id_event():
+    api_gateway_proxy_event = {
+        "queryStringParameters": {"patientId": "9000AB0009"},
+    }
+    return api_gateway_proxy_event
+
+
+@pytest.fixture
+def context():
+    @dataclass
+    class LambdaContext:
+        function_name: str = "test"
+        aws_request_id: str = "88888888-4444-4444-4444-121212121212"
+        invoked_function_arn: str = (
+            "arn:aws:lambda:eu-west-1:123456789101:function:test"
+        )
+
+    return LambdaContext()
```

Comment:
Extract recurring pytest fixtures to a `conftest.py` file

Reply:
Done